src/main/java/org/fcrepo/client/FcrepoClient.java

```diff
@@ -153,14 +166,25 @@ public HistoricMementoBuilder createMemento(final URI url, final String mementoD
     }
 
     /**
-     * Interact with the Transaction API (rewrite this comment)
+     * Interact with the Transaction API - start, commit, etc
      *
      * @return a transaction request builder object
      */
     public TransactionBuilder transaction() {
         return new TransactionBuilder(this);
     }
 
+    /**
+     * Create a new {@link TransactionalFcrepoClient} which adds the
+     * {@link org.fcrepo.client.FcrepoResponse.TransactionURI} to each request
+     *
+     * @param uri the Transaction to add to each request
+     * @return a TransactionFcrepoClient
+     */
+    public TransactionalFcrepoClient transactionalClient(final FcrepoResponse.TransactionURI uri) {
```

Comment:
It would be good to add a test of this method. I am also wondering if it would make sense to add a `startTransactionalClient` method to the `TransactionBuilder`, which would call start and then return a new TransactionalClient with the URI of the new transaction, since it seems like almost every usage of `fcrepoClient.transactionalClient` would require those same steps.

Reply:
I was thinking about a way to create it from the `TransactionalBuilder` as well. At the moment I'm playing around with the idea of adding the `startTxClient` inside of the `RequestBuilder` from `txBuilder.start()`, though I don't know if adding an additional method of execution is a good idea since perform is already defined. Otherwise maybe the `startTxClient` could take a Map of headers so they can be set before execution.